infobip-typescript-generator-extension-model-showcase/src/main/java/com/infobip/typescript/showcase/hierarchy/HierarchyTypeScriptFileGenerator.java

```diff
@@ -47,4 +48,14 @@ public Path outputFilePath(Path basePath) {
 
         return lib.resolve("Hierarchy.ts");
     }
+
+    @Override
+    protected CustomValidationSettings getCustomValidationSettings() {
```

Comment:
kod ove i ostalih validacija mozda bi imalo smisla da settingsi za custom anotacije i dekoratore budu Optional jer nisu potrebni za sve use caseove (pogotovo one jednostavnije)
Takodjer mozda ovu metodu preimenovati u getCustomValidationAnnotationSettings i onu dolje u getCustomDecoratorBasePath, a mozda i te settingse na neki nacin ujednaciti da obje vracaju path ili da obje vracaju object wrapper koji sadrzi path

Reply:
stavljeno da bude optionala

Reply:
renameano, ali ne bih ih wrappao i ili izmjenjivao to bi ya sad ostalo as is